server/constants/messages.go

```diff
@@ -54,6 +54,7 @@ const (
 	ErrorFetchProjectList                        = "Error in fetching project list"
 	ErrorDecodingBody                            = "Error in decoding body"
 	ErrorCreateTask                              = "Error in creating task"
+	ErrorCreateSubscription                      = "Error in creating subscription"
```

Comment:
```suggestion
	ErrorCreateSubscription                      = "Error in creating the subscription"
```

Reply:
I think we should not add `the` here as we are not specifying the subscription details with the error message